addons/account/models/account_journal.py

```diff
@@ -884,15 +884,14 @@ def _get_journal_outbound_outstanding_payment_accounts(self):
             account_ids.add(line.payment_account_id.id or self.company_id.account_journal_payment_credit_account_id.id)
         return self.env['account.account'].browse(account_ids)
 
-    # TODO move to `account_reports` in master
```

Comment:
Is this used?

Reply:
yes, I am using it in `open_bank_difference_action` below
so no need to move it to master

Reply:
It would be better to make a field out of the batch computation below
https://github.com/odoo/odoo/pull/129569/files#diff-6e3139ca3c848ecd9159cbee86a94aa51a3c7e0d2b62bbccfac1023c18a1edf5R340
to remove this less optimized version

Reply:
done, method is unused